capstone-frontend/src/components/Navbar.jsx

```diff
@@ -58,7 +58,12 @@ class Navbar extends Component {
           <ul className="navbar-nav">
             <form className="form-inline my-2 my-lg-0" onSubmit={this.handleSearchInput}>
               <input className="form-control mr-sm-2" id="search" type="search" placeholder="Search" aria-label="Search" />
-              <button type="submit" style={{backgroundColor: "transparent", border: "none"}}><img src="/images/search_icon.png" alt="Search Icon"/></button>
+              <button type="submit" style={{ backgroundColor: "transparent", border: "none" }}>
+                <svg width="1.5em" height="1.5em" viewBox="0 0 16 16" className="bi bi-search" fill="white" xmlns="http://www.w3.org/2000/svg">
+                  <path fillRule="evenodd" d="M10.442 10.442a1 1 0 0 1 1.415 0l3.85 3.85a1 1 0 0 1-1.414 1.415l-3.85-3.85a1 1 0 0 1 0-1.415z" />
```

Comment:
Not sure exactly what this does, can we discuss at meeting?

Reply:
It adds a Search Icon